src/store/ducks/opensrp/students/index.ts

```diff
@@ -0,0 +1,146 @@
+import { keyBy, values } from 'lodash';
```

Comment:
Can we please NOT call this "students".  This is a "clients" reducer unless I am wrong.  Lets not mention students here.